CONTRIBUTING.md

```diff
@@ -26,16 +26,16 @@ Config branches are entirely separate from the `main` history in this repository
 
 ### Brand new configuration
 
-If you are creating a brand new configuration, and don't have the config stored in another repository, just checkout a `dev-*` branch from `main` and delete everything except `.github/workflows/pr-1-ci.yml`, `.github/workflows/pr-3-bump-tag.yml` and `.github/workflows/validate-json.yml`, then add your config.
+If you are creating a brand new configuration, and don't have the config stored in another repository, just checkout a `dev-*` branch from `main` and delete everything except `.github/workflows/ci.yml` and `.github/workflows/schedule.yml`, then add your config.
 
 ### Config is Stored in Another Repository
 
-Create a `dev-*` branch by adding the config repository as a remote and checking out the config branch: 
+Create a `dev-*` branch by adding the config repository as a remote and checking out the config branch:
 
 ```bash
 git remote add <config_repo> <config_repo_url>  # ex. git remote add config git@github.com/my/configs.git
 git checkout <config_repo>/<config_branch> -b dev-<config_name>  # checkout config from new remote + add to branch, ex. git checkout config/main -b dev-1deg_abc_def
-git checkout main -- .github/workflows/call-*.yml .github/workflows/validate-json.yml  #
+git checkout main -- .github/workflows/ci.yml .github/workflows/schedule.yml
```

Comment:
Same question regarding schedule.  `call-*` is also referenced earlier in the file at line 5

Reply:
Correct, removed in https://github.com/ACCESS-NRI/access-om2-configs/pull/117/commits/04002d09eca489a2112c86e609242dc950d214f8